contracts/proposal/dao-proposal-single/src/testing/contracts.rs

```diff
@@ -39,16 +39,16 @@ pub(crate) fn cw20_stake_contract() -> Box<dyn Contract<Empty>> {
     Box::new(contract)
 }
 
-pub(crate) fn v1_proposal_single_contract() -> Box<dyn Contract<Empty>> {
-    let contract = ContractWrapper::new(
-        cw_proposal_single_v1::contract::execute,
-        cw_proposal_single_v1::contract::instantiate,
-        cw_proposal_single_v1::contract::query,
-    )
-    .with_reply(cw_proposal_single_v1::contract::reply)
-    .with_migrate(cw_proposal_single_v1::contract::migrate);
-    Box::new(contract)
-}
+// pub(crate) fn v2_proposal_single_contract() -> Box<dyn Contract<Empty>> {
+//     let contract = ContractWrapper::new(
+//         dao_proposal_single_v2::contract::execute,
+//         dao_proposal_single_v2::contract::instantiate,
+//         dao_proposal_single_v2::contract::query,
+//     )
+//     .with_reply(dao_proposal_single_v2::contract::reply)
+//     .with_migrate(dao_proposal_single_v2::contract::migrate);
+//     Box::new(contract)
+// }
```

Comment:
At some point it will be when we want to do contract migrations and tests, but let's kill it for now.

Reply:
> At some point it will be when we want to do contract migrations and tests, but let's kill it for now.

wait we should definitely be doing contract migration and tests. i'm confused, we've been doing migrations and tests the whole time right? then we shouldn't delete this.